Dockerfile.prod

```diff
@@ -0,0 +1,21 @@
+FROM node:12
+
+RUN mkdir -p /app
+WORKDIR /app
+
+# copy the project code
+COPY src /app/src
+COPY static /app/static
+COPY .env .eslintrc package.json package-lock.json rollup.config.js /app/
+
+# install our dependencies
+RUN npm install
+
+# Build the project
+RUN npm run build
+
+# expose the server port
+EXPOSE 3000
```

Comment:
Isn't the port listed in `process.env.PORT`?

Reply:
This line doesn't affect the image at all. It simply acts as documentation that this container should have the port 3000 mapped to something. I can add `ENV PORT 3000` to set it if it wasn't passed in the environment or the .env file, but it seems redundant since it defaults to it anyways